src/makeCalculator.js

```diff
@@ -37,7 +37,25 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(func, num) {
+      this.result = func(this.result, num);
+
+      return this;
+    },
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
```

Comment:
The 'divide' method does not handle the case when 'b' is zero. Dividing by zero in JavaScript will return Infinity, which may not be the expected behaviour.